nodes/config/tts.html

```diff
@@ -111,3 +76,468 @@ <h3>Details</h3>
   <p><code>Name</code> property is shown on selection lists to easily switch between different text-to-speech
      configurations.</p>
 </script>
+
+<script type="text/javascript">
+  function languages() {
+    return [
```

Comment:
Would it be okay to play with the data structure here?

We can have something similar as below;

```
{
  "arb": {
    "code": "arb",
    "name": "Arabic",
    "voices": [
      {
        "id": "Zeina",
        "name": "Zeina",
        "gender": "Female"
      }
    ]
  },
  "cmn-CN": {
    "code": "cmn-CN",
    "name": "Chinese Mandarin",
    "voices": [
      {
        "id": "Zhiyu",
        "name": "Zhiyu",
        "gender": "Female"
      }
    ]
  },
  "cy-GB": {
    "code": "cy-GB",
    "name": "Welsh",
    "voices": [
      {
        "id": "Gwyneth",
        "name": "Gwyneth",
        "gender": "Female"
      }
    ]
  }
  // the rest comes here
}
```

This will allow us to get the language relevant data without looping it through as `languages[selectedLanguage.code]`.